[OSDOCS-11832] OCM book for HCP topic map (#84429)
openshift/openshift-docs: modules/ocm-networking-tab-concept.adoc

[id="ocm-networking-tab_{context}"]
= Networking tab

> The **Networking** tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router can be made private by selecting the respective box below each of them. If applicable, you can also find your virtual private cloud (VPC) details on this tab.

Reviewer: ```suggestion
The **Networking** tab provides a control plane API endpoint as well as the default application router. Both the control plane API endpoint and the default application router can be made private by selecting the respective box below each title. If applicable, you can also find your virtual private cloud (VPC) details on this tab.
```
Not sure if "title" or "label" or something else entirely would be the best word to substitute here. I just think "each of them" is a little vague.

ifdef::openshift-rosa[]

> Select **Edit application ingress** button to edit the existing application ingress. You can change your application ingress to private or public by checking or unchecking the "Make router private" checkbox as desired.

Reviewer: ```suggestion
Select the **Edit application ingress** button to edit the existing application ingress. You can change your application ingress to private or public by checking or unchecking the "Make router private" checkbox.
```

endif::openshift-rosa[]
ifdef::openshift-rosa-hcp[]

> You can change your application ingress to private or public by select the **Edit application ingress** button then checking or unchecking the "Make router private" checkbox as desired.

Reviewer: ```suggestion
You can change your application ingress to private or public by selecting the **Edit application ingress** button then checking or unchecking the "Make router private" checkbox.
```
